pkg/build/node/cni.go

```diff
@@ -97,6 +97,12 @@ rules:
     verbs:
       - list
       - watch
+  - apiGroups:
```

Comment:
just unrelated to this change but can you remove the apparmor profiles?
```
    apparmor.security.beta.kubernetes.io/allowedProfileNames: runtime/default
    apparmor.security.beta.kubernetes.io/defaultProfileName: runtime/default	    apparmor.security.beta.kubernetes.io/defaultProfileName: runtime/default
```

Reply:
we probably shouldn't? people are still trying to get psp to work